src/Repository/WishlistRepository.php

```diff
@@ -140,4 +188,17 @@ public function findOneByShopUserAndName(ShopUserInterface $shopUser, string $na
             ->getOneOrNullResult()
             ;
     }
+
+    public function deleteWishlistsNotModifiedSince(\DateTimeInterface $period): void
+    {
+        $this
+            ->createQueryBuilder('o')
+            ->delete()
+            ->where('o.updatedAt IS NULL AND o.createdAt < :period')
```

Comment:
you're setting the updatedAt property in the Wishlist constructor. So IMO this property is never null

Reply:
Hi @mcsky and thank you for the review!

No, I'm not setting the `updatedAt` in the constructor. Did I misunderstand your comment? Maybe you're suggesting me to set it?